src/components/bulkConcepts/container/BulkConceptsPage.jsx

```diff
@@ -61,10 +61,11 @@ export class BulkConceptsPage extends Component {
       target: { name },
     } = event;
     const targetName = name.split(',');
+    const query = `q=${this.state.searchInput}`;
     if (targetName[1] === 'datatype') {
-      this.props.addToFilterList(targetName[0], 'datatype');
+      this.props.addToFilterList(targetName[0], 'datatype', query);
```

Comment:
It'll be nice to destructure addToFilterList since you are using in more than one place.

Reply:
Thanks for the feedback. However, it can't be used here as `addToFilterList` has already been declared in the upper scope.

Reply:
Alright then.